src/users/api/UserDto.ts

```diff
@@ -31,6 +31,9 @@ export class UserDto {
   @Exclude()
   createdAt: Date;
 
+  @ApiHideProperty()
+  toJSON: any;
```

Comment:
You should use additional properties instead, otherwise, the DAST tools will struggle with the challenge to find this vulnerability)